time/src/time-driver.ts

```diff
@@ -9,6 +9,8 @@ import {makeThrottleAnimation} from './throttle-animation';
 import {runVirtually} from './run-virtually';
 import {TimeSource} from './time-source';
 const requestAnimationFrame = require('raf');
+const now = require('performance-now');
```

Comment:
`perfomance-now` is based on `process.hrtime`, so I would only use it in the mock time source (the one that will run most likely in node)